### DISCUSSION TITLE
Filament Spatie Media Library file names

### DISCUSSION THREAD

**Discussion by neflictus** [filamentphp/filament#5769] [Help]
How can I change the default filename for the uploaded files when using the SpatieMediaLibrary Filament plugin ?

`SpatieMediaLibraryFileUpload::make('thumbnail')`
`   ->collection('cars')`
`   ->columnSpan(4)`
`   ->multiple()`
`   ->responsiveImages()`
`   ->enableReordering()`
`   ->label(__('Image')),`

**Comment by danharrin** [comment]
You'll need to configure that inside the media library package, not Filament code

**Comment by annya-a** [comment]
I was able to set custom title for image with the help of such code:
```
SpatieMediaLibraryFileUpload::make('image')
    ->collection('image')
    ->getUploadedFileNameForStorageUsing(function (BaseFileUpload $component, TemporaryUploadedFile $file) {
        return 'my-title.' . $file->getClientOriginalExtension();
    }),
```
